src/HtmlGenerator.Tests/GitBlameVisitorTests.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using GitGlyph;
+using LibGit2Sharp;
+using Microsoft.SourceBrowser.HtmlGenerator.Utilities;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace HtmlGenerator.Tests
+{
+    [TestClass]
+    public class GitBlameVistorTests
+    {
+        [TestMethod]
+        public void TestVisit()
+        {
+            string pathToTempDirectory = Path.Combine(Path.GetTempPath(),"GitBlameVisitorRepo\\");

```

Comment:
style: add a space after the comma here, and at lines 18 and 23.
The last \\ is not necessary also.